src/render_cmd.rs

```diff
@@ -670,4 +675,37 @@ mod tests {
         assert_eq!(stage_eu_w4_deployment_expected_helm_cmd, got_stage.1);
         assert_eq!(res.commands.len(), 3);
     }
+
+    #[test]
+    fn pipe_output_through_tool() {
```

Comment:
Please also add *unit* test cases for:
- multiple `--pipe` usages
- an error case, for example piping into a tool that doesn't exist

And I'd also propose an integration test. The `grep` example used in the first unit tests is a great choice.